onepassword/client.py

```diff
@@ -444,10 +446,10 @@ def get_item(uuid: str | bytes, fields: str | bytes | list | None = None):
         elif isinstance(fields, str):
             item = {
                 fields: read_bash_return(
-                    "op item get {} --fields label={}".format(uuid, fields), single=False).rstrip('\n')
+                    "op item get {} --fields label={} {}".format(uuid, fields, vault_flag), single=False).rstrip('\n')
             }
         else:
-            item = json.loads(read_bash_return("op item get {} --format=json".format(uuid), single=False))
+            item = json.loads(read_bash_return("op item get {} --format=json {}".format(uuid, vault_flag), single=False))
```

Comment:
Same as above plus what if it's empty?
```suggestion
            item = json.loads(read_bash_return("op item get {} --format=json --vault {}".format(uuid, vault_flag), single=False))
```